dataservice/api/biospecimen/models.py

```diff
@@ -0,0 +1,78 @@
+           specimen analyte
+    :param concentration: The concentration of an analyte or aliquot extracted
+     from the biospecimen or biospecimen portion, measured in
+     milligrams per milliliter
+    :param volume: The volume in microliters (ml) of the aliquots derived from
+     the analyte(s) shipped for sequencing and characterization
+    :param shipment_date: The date item was shipped in YYYY-MM-DD format
+    :param uberon_id: The ID of the term from Uber-anatomy ontology
+     which represents harmonized anatomical ontologies
+    """
+    __tablename__ = 'biospecimen'
+    __prefix__ = 'BS'
+
+    external_sample_id = db.Column(db.Text(),
+                                   doc='Name given to sample by contributor')
+    external_aliquot_id = db.Column(db.Text(),
+                                    doc='Name given to aliquot by contributor')
+    tissue_type = db.Column(db.Text(),
+                            doc='Description of the kind of biospecimen '
+                            'collected')
+    composition = db.Column(db.Text(),
+                            doc='The cellular composition of the biospecimen')
+    anatomical_site = db.Column(db.Text(),
+                                doc='The anatomical location of collection')
+    age_at_event_days = db.Column(db.Integer(),
+                                  doc='Age at the time of event occurred in '
+                                      'number of days since birth.')
+    tumor_descriptor = db.Column(db.Text(),
+                                 doc='Disease present in the biospecimen')
+    shipment_origin = db.Column(db.Text(),
+                                doc='The original site of the aliquot')
+    shipment_destination = db.Column(db.Text(),
+                                     doc='The site recieving the aliquot')
+    analyte_type = db.Column(db.Text(), nullable=False,
+                             doc='The molecular description of the aliquot')
+    concentration = db.Column(db.Float(),
+                              doc='The concentration of the aliquot')
+    volume = db.Column(db.Float(),
+                       doc='The volume of the aliquot')
+    shipment_date = db.Column(db.DateTime(),
+                              doc='The date the aliquot was shipped')
+    uberon_id = db.Column(db.Text(),
+                          doc='The ID of the term from Uber-anatomy ontology'
+                          'which represents harmonized anatomical ontologies')
+    participant_id = db.Column(KfId(),
+                               db.ForeignKey('participant.kf_id'),
+                               nullable=False,
+                               doc='The kf_id of the biospecimen\'s donor')
+    genomic_files = db.relationship(GenomicFile,
+                                    cascade="all, delete-orphan",
+                                    backref=db.backref(
+                                        'genomic_files', lazy=True),
```

Comment:
The backref should be named `biospecimen` to represent the many to one going from genomic file to biospecimen (this will give us GenomicFile.biospecimen).

Reply:
👍